velox/functions/prestosql/tests/DecimalArithmeticTest.cpp

```diff
@@ -378,6 +378,30 @@ TEST_F(DecimalArithmeticTest, round) {
           {UnscaledLongDecimal::max().unscaledValue(),
            UnscaledLongDecimal::min().unscaledValue()},
           DECIMAL(38, 1))});
+
+  // Min and max short decimals.
+  testDecimalExpr<TypeKind::SHORT_DECIMAL>(
+      {makeShortDecimalFlatVector(
+          {UnscaledShortDecimal::max().unscaledValue(),
+           UnscaledShortDecimal::min().unscaledValue()},
+          DECIMAL(15, 0))},
+      "round(c0)",
+      {makeShortDecimalFlatVector(
+          {UnscaledShortDecimal::max().unscaledValue(),
+           UnscaledShortDecimal::min().unscaledValue()},
+          DECIMAL(15, 0))});
+
+  // Min and max long decimals.
+  testDecimalExpr<TypeKind::LONG_DECIMAL>(
```

Comment:
There is min and max long decimals at L378. Do we still need this.

Reply:
The tests are for the max values for short and long decimals with no scale to check whether there might be an overflow happening depending on the calculation. I think L378 tests verify DECIMAL(38,1), these tests check DECIMAL(38,0).

Reply:
Ok, would you pls mark it ready for review.